src/components/ContactsView.js

```diff
@@ -1,23 +1,35 @@
-import { useEffect, useState } from "react"
-import { useParams } from "react-router-dom"
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { Link } from "react-router-dom";
 
 function ContactsView() {
-  const [contact, setContact] = useState(false)
+  const [contact, setContact] = useState(false);
 
-  //TODO: Get the contact to load from the params and fetch.
-  //With useEffect, load the contact when params changes
-  //and update contact state
+  const params = useParams();
+
+  useEffect(() => {
+    fetch(`http://localhost:4000/contacts/${params.id}`)
```

Comment:
What does ```${params.id}``` do?

Reply:
adds the specific id of the contact to the url